### PR TITLE
Ensure compaction is executed at even intervals to avoid overload

### PR DESCRIPTION
**Pull request by serathius**
Should reduce number of flakes due to etcd being overloaded by too many compaction at once

### CODE REVIEW

**Comment by serathius** [comment]
cc @jmhbnz

File: tests/robustness/traffic/traffic.go [hunk 1]
```diff
@@ -181,5 +197,33 @@ func (p Profile) WithoutCompaction() Profile {
 type Traffic interface {
 	Run(ctx context.Context, c *client.RecordingClient, qpsLimiter *rate.Limiter, ids identity.Provider, lm identity.LeaseIDStorage, nonUniqueWriteLimiter ConcurrencyLimiter, finish <-chan struct{})
 	ExpectUniqueRevision() bool
-	WithoutCompact() Traffic
+}
+
+func RunCompactLoop(ctx context.Context, c *client.RecordingClient, period time.Duration, finish <-chan struct{}) {
+	var lastRev int64 = 2
```
Comment (jmhbnz) [inline tests/robustness/traffic/traffic.go:203]: Why is `lastRev` always set to 2 here?

Comment (serathius) [reply tests/robustness/traffic/traffic.go:203]: It's initial value.

Comment (jmhbnz) [reply tests/robustness/traffic/traffic.go:203]: Ahh there is only one possible compaction loop per simulation. I understand now thanks.

**Review by jmhbnz — APPROVED** [review]
LGTM

**Pull request merged**